### PR TITLE
Fix call events layout for message bubble

### PR DESCRIPTION
**Pull request by gsouquet**
Fixes vector-im/element-web#18144

### CODE REVIEW

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
Screenshot_20210723_170730

This doesn't seem to be really solving the issue for me :thinking: The call event tiles should probably span over a big part of the timeline.

File: res/css/views/rooms/_EventBubbleTile.scss [hunk 1]
```diff
@@ -230,6 +230,12 @@ limitations under the License.
         }
     }
 
+    &.mx_EventTile_bubbleContainer {
+        .mx_EventTile_info {
```
Comment (gsouquet) [inline res/css/views/rooms/_EventBubbleTile.scss:234]:
```suggestion
        .mx_EventTile_line,
        .mx_EventTile_info {
```

**Comment by gsouquet** [comment]
> This doesn't seem to be really solving the issue for me 🤔 The call event tiles should probably span over a big part of the timeline.

Looks like different replies have different DOM structures, I added an extra selector that does seem to help here!

**Comment by SimonBrandner** [comment]
@gsouquet, I think that if you change 

https://github.com/matrix-org/matrix-react-sdk/blob/dbe24fab70bcebdc789a0f8a9865db28339d6203/res/css/views/messages/_CallEvent.scss#L26 

to 

```
width: 75%; 
```

We should actually achieve the effect of the tiles having the same width

**Review by SimonBrandner — APPROVED** [review]
Thank you!

**Review by turt2live — APPROVED** [review]
Screenshots in the PR description are appreciated for these sorts of changes to aide review and to be a reference point down the line when we're trying to debug why it broke or where it broke.

Please include screenshots :)

**Comment by gsouquet** [comment]
Thank you for the review, this fixes a regression and does not bring any new feature nor changes in design

Screen Shot 2021-07-30 at 09 07 59

**Pull request merged**